misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobQueue.kt

```diff
@@ -97,6 +103,102 @@ internal class SqsJobQueue @Inject internal constructor(
     }
   }
 
+  override fun batchEnqueue(
```

Comment:
does it make sense to make single enqueue just delegate to this?

Reply:
Possibly, but the batch api has a different contract than the single one. It doesn't throw. Not sure how to make it look the same (or if we should), in case callers of this were catching specific dynamo errors..

Reply:
discussed offline: Batch enqueue will throw an exception that contains the list of failed items